dvc/analytics.py

```diff
@@ -153,7 +153,7 @@ def _find_or_create_user_id():
 
     # Since the `fname` and `lockfile` are under the global config,
     # we need to make sure such directory exist already.
-    makedirs(config_dir, exist_ok=True)
+    makedirs(config_dir, exist_ok=True, mode=0o755)
```

Comment:
We shouldn't hardcode the permissions, we should use the default ones with the umask.